sycl/source/detail/memory_manager.cpp

```diff
@@ -244,6 +244,40 @@ void *MemoryManager::allocateMemSubBuffer(ContextImplPtr TargetContext,
   return NewMem;
 }
 
+struct term_positions {
```

Comment:
Could you please follow the same guidelines for naming entities which are already used in the file?
term_positions -> TermPositions
x_term -> XTerm

Reply:
done